tests/system_tests/test_core/test_wandb_run.py

```diff
@@ -178,7 +178,6 @@ def test_network_fault_graphql(wandb_backend_spy):
     with wandb_backend_spy.freeze() as snapshot:
         uploaded_files = snapshot.uploaded_files(run_id=run.id)
 
-        assert "wandb-metadata.json" in uploaded_files
```

Comment:
This used to work because there were two places where metadata was generated: one from settings/git state, and another where system resources were probed. the first step is fast, while the second takes some time. I combined the steps and moved them into the system monitor, so this doesn't always succeed because we generate and upload this file on a best-effort basis.